### ISSUE TITLE
이거 양자화 되나요?

### ISSUE THREAD

**Issue by skku970412** [rebellions-sw/optimum-rbln#316]
안녕하세요
AI칩콘참여중인
이재욱 기계공학부 성균관대입니다.

모델양자화해서
추론코드실행하면

python inspect_logits_raw.py 2025-10-02 01:38:58,459 INFO [rebel-compiler] Load model completed. Elasped time: 0:00:01 2025-10-02 01:38:58,909 INFO [rebel-compiler] Load model completed. Elasped time: 0:00:00 Top-20 raw logits (id : value) : 198 Ċ 118.5 271 ĊĊ 65.75 201 č 49.1875 715 ĠĊ 48.8125 319 čĊ 45.3125 220 Ġ 42.9375 624 .Ċ 42.3125 9338 ...Ċ 40.1875 345 ,Ċ 38.9375 151643 <|endoftext|> 37.75 510 :Ċ 37.4375 11 , 36.875 13 . 34.875 6913 -Ċ 34.3125 320 Ġ( 34.1875 1406 ĊĊĊ 32.75 340 )Ċ 31.46875 12 - 31.4375 16 1 30.28125 5267 ?Ċ 29.59375 top1 - top2 raw logit diff: 52.75 logits min, max, mean, std: -42.0625 118.5 -18.644044876098633 6.162200450897217 Saved raw_logits_report.json

Top-20 raw logits
id=198 로짓 = 118.5. 2위는 65.75. top1 - top2 = 52.75 → softmax에서 top1 확률이 1거의 1이라

한단어만계속나오는문제가 있어서요

컴파일코드는 아래입니다.

import os

from optimum.rbln import RBLNAutoModelForCausalLM

def main():
    model_id = "Qwen/Qwen3-0.6B"

    # Compile and export
    model = RBLNAutoModelForCausalLM.from_pretrained(
        model_id=model_id,
        export=True,  # export a PyTorch model to RBLN model with optimum
        rbln_batch_size=1,
        rbln_max_seq_len=8192,
        rbln_quantization={"weights": "int8",}  # ← 여기서 W8A8 지정

    )

    # Save compiled results to disk
    model.save_pretrained(os.path.basename(model_id))

if __name__ == "__main__":
    main()

감사합니다.
